src/Arrow/index.tsx

```diff
@@ -37,7 +38,7 @@ function useArrowStyles(primaryPlacement, arrowSize) {
 	return arrowStyles;
 }
 
-const Arrow = forwardRef((props, ref) => {
+const Arrow = forwardRef((props: {placement: string, size: number, distanceFromEdge: number, style}, ref: React.RefObject<HTMLElement>) => {
```

Comment:
The `placement` prop can be defined more granularly based on the `PLACEMENTS` constant – or maybe you can even import that type from popperJS. Should result in a nice auto-complete for this prop, as only a limited number of strings are actually allowed as values.